### PR TITLE
Fixed ClassCastException in describe() of BatchPortal.

### PR DESCRIPTION
**Pull request by matriv**
The issue was discovered by new crate-qa test:
https://github.com/crate/crate-qa/pull/51/files#diff-4ebb99449d934c2947c0fbc26ced5d93

### CODE REVIEW

**Comment by matriv** [comment]
@mfussenegger Please take a look at the new integTest. This test now fails with the proper exception. Before that we got the class cast exception... Do you think it's ok like that or would a UnitTest on BatchPortal be better?

File: sql/src/test/java/io/crate/integrationtests/PostgresITest.java [hunk 1]
```diff
@@ -689,6 +707,19 @@ public void testCountDistinctFromJoin() throws Exception {
         }
     }
 
+    @Test
+    public void testDescribeOnBatchPortal() throws Exception {
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: Can we add a comment or clarify the test name further? It's not obvious how this test works or what it does

File: sql/src/test/java/io/crate/integrationtests/PostgresITest.java [hunk 2]
```diff
@@ -531,6 +531,24 @@ public void testExecuteBatchWithDifferentStatements() throws Exception {
         }
     }
 
+    @Test
+    public void testExecuteBatchWithSelectStatement() throws Exception {
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: maybe rename to something like `test_execute_batch_fails_with_read_operations`

File: sql/src/test/java/io/crate/integrationtests/PostgresITest.java (hunk 1 above)
Comment (matriv) [reply sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: This is not needed, sorry, will remove

File: CHANGES.txt [hunk 3]
```diff
@@ -122,3 +122,5 @@ Changes
 Fixes
 =====
 
+- Fixes an issue that would cause a ClassCastException to be thrown when using
```
Comment (mfussenegger) [inline CHANGES.txt]: Maybe clarify that an error case raised a ClassCastException and that it's now returning a sensible error.

Comment (matriv) [reply CHANGES.txt]: Sure, Fixup pushed.

**Pull request merged**